libs/application/templates/financial-aid/src/lib/FinancialAidTemplate.ts

```diff
@@ -61,7 +61,15 @@ const FinancialAidTemplate: ApplicationTemplate<
           ],
         },
         on: {
-          SUBMIT: [{ target: ApplicationStates.DRAFT, cond: () => true }],
+          SUBMIT: [
+            {
+              target: ApplicationStates.DRAFT,
+              cond: hasActiveCurrentApplication,
+            },
+            {
+              target: ApplicationStates.SUBMITTED,
```

Comment:
Just wondering. In what situation is the "target: ApplicationStates.SUBMITTED," triggered?